packages/guideline-blocks-settings/src/components/BlockItemWrapper/types.ts

```diff
@@ -1,8 +1,10 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { MenuItemStyle } from '@frontify/fondue';
+import { ReactElement } from 'react';
 
 export type BlockItemWrapperProps = {
+    children: ReactElement;
```

Comment:
```suggestion
    children: ReactNode;
```

To allow `string` otherwise feel free to keep it as is